Header header
int16 WallSensorInMM
int16 BatteryVoltageInmV
int16 LeftDropInMM
int16 RightDropInMM
int8 RightMagSensor
int8 LeftMagSensor
bool SNSR_DC_JACK_CONNECT
bool SNSR_DUSTBIN_IS_IN
bool SNSR_LEFT_WHEEL_EXTENDED
bool SNSR_RIGHT_WHEEL_EXTENDED
bool LSIDEBIT
bool LFRONTBIT
bool RSIDEBIT
bool RFRONTBIT
bool BTN_SOFT_KEY
bool BTN_SCROLL_UP
bool BTN_SCROLL_DOWN
bool BTN_START
bool BTN_BACK
int32 LeftWheel_Encoder
int32 LeftWheel_PositionInMM
int16 LeftWheel_RPM
int32 RightWheel_Encoder
int32 RightWheel_PositionInMM
int16 RightWheel_RPM
int32 Laser_Encoder
int16 Laser_RPM
int8 FuelPercent
bool ChargingActive
bool ChargingEnabled
bool ConfidentOnFuel
bool OnReservedFuel
bool EmptyFuel
bool BatteryFailure
